service/src/test/java/greencity/mapping/EcoNewsCommentVOMapperTest.java

```diff
@@ -0,0 +1,48 @@
+package greencity.mapping;
+
+import greencity.ModelUtils;
+import greencity.dto.econewscomment.EcoNewsCommentVO;
+import greencity.entity.EcoNews;
+import greencity.entity.EcoNewsComment;
+import greencity.entity.User;
+import java.util.stream.Collectors;
+import static org.junit.jupiter.api.Assertions.*;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.InjectMocks;
+import org.springframework.test.context.junit.jupiter.SpringExtension;
+
+@ExtendWith(SpringExtension.class)
```

Comment:
Use **MockitoExtension** instead of **SpringExtension** because you don't need to involve here Spring for testing.

Reply:
ty, changed